RELEASING.md

```diff
@@ -0,0 +1,49 @@
+# Release Process
+
+## Versioning
+The versioning follows the process described at [semver.org](https://semver.org/).
+
+### Public API
+We consider the public API of this project to be whatever is included in the
+`LIBRARY_PUBLIC_HEADERS` defined in `src/CMakeLists.txt`. Currently, this includes
+all the header files in `src/mococrw/`. Changes to the files will be versioned according
+to semantic versioning scheme.
+
+## PRs & Releasing
+
+### PR Checklist
+(probably to be extended)
```

Comment:
Remove this comment. If we want to extend this, we will, if we don't want to, we'll have this comment sitting there that would no longer be correct.